diff -ur libixion-0.17.0.orig/configure.ac libixion-0.17.0/configure.ac
--- libixion-0.17.0.orig/configure.ac	2023-05-16 22:03:36.515096324 +0300
+++ libixion-0.17.0/configure.ac	2023-05-16 22:03:48.145096070 +0300
@@ -127,7 +127,7 @@
 	AC_DEFINE([NDEBUG], [], [Disable debugging information])
 ])
 
-PKG_CHECK_MODULES([MDDS],[mdds-2.0 >= 1.9.99])
+PKG_CHECK_MODULES([MDDS],[mdds-2.1 >= 2.1.0])
 
 AC_ARG_ENABLE(log-debug,
 	AS_HELP_STRING([--enable-log-debug], [Enable debug outputs.])],
